securing_apps/topics/client-registration.adoc

```diff
@@ -32,6 +32,7 @@ The bearer token can be issued on behalf of a user or a Service Account. The fol
 
 If you are using a bearer token to create clients it's recommend to use a token from a Service Account with only the `create-client` role (see link:{adminguide_link}[{adminguide_name}] for more details).
```

Comment:
change "create clients it's recommend to use..." to "create clients, we recommend that you use.."